src/main/java/net/earthcomputer/clientcommands/command/arguments/PacketTypeArgument.java

```diff
@@ -90,7 +91,8 @@ private static PacketTypes get() {
             var clientbound = packetTypesCache.computeIfAbsent(decoder, k -> getPacketTypes(decoder.protocolInfo));
             var encoder = (PacketEncoder<?>) pipeline.get("encoder");
             var serverbound = packetTypesCache.computeIfAbsent(encoder, k -> getPacketTypes(encoder.protocolInfo));
-            return new PacketTypes(clientbound, serverbound);
+            var c2cbound = packetTypesCache.computeIfAbsent("xpple", k -> getPacketTypes(C2CPacketHandler.C2C));
```

Comment:
Maybe something a bit more descriptive like `"c2c"`?

Reply:
It was a dummy key but sure.